docs/content/docs/connectors/pipeline-connectors/postgres.md

```diff
@@ -0,0 +1,398 @@
+---
+title: "Postgres"
+weight: 2
+type: docs
+aliases:
+- /connectors/pipeline-connectors/Postgres
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Postgres Connector
+
+Postgres connector allows reading snapshot data and incremental data from Postgres database and provides end-to-end full-database data synchronization capabilities.
+This document describes how to setup the Postgres connector.
+Note: Since the Postgres WAL log cannot parse table structure change records, Postgres CDC Pipeline does not support table structure changes.
```

Comment:
> Postgres CDC Pipeline does not support table structure changes.

Postgres CDC Pipeline Source does not support synchronizing table structure changes currently.